gitlab/datadog_checks/gitlab/metrics.py

```diff
@@ -7,7 +7,7 @@
     "gitlab_banzai_cacheless_render_real_duration_seconds": "banzai.cacheless_render_real_duration_seconds",
     "gitlab_cache_misses_total": "cache.misses_total",
     "gitlab_cache_operation_duration_seconds": "cache.operation_duration_seconds",
-    "gitlab_cache_operations_total": "cache_operations_total",
+    "gitlab_cache_operations_total": "cache.operations_total",
```

Comment:
This metric doesn't follow the convention, but the renaming is a breaking change.

Reply:
Yes, I was wondering if it would be better to change it to match the metadata.csv and the convention, but I think you're right- it's small enough that just changing the metadata.csv to match should be fine.

Reply:
Updated!